mongodb-legacy.d.ts

```diff
@@ -206,8 +206,8 @@ declare class LegacyChangeStream<TSchema extends Document = Document, TChange ex
   /**
    * Try to get the next available document from the Change Stream's cursor or `null` if an empty batch is returned
    */
-  tryNext(): Promise<Document | null>;
-  tryNext(callback: Callback<Document | null>): void;
+  tryNext(): Promise<TChange | null>;
```

Comment:
This is unrelated to typescript's update.  This fails even with ts@5.0.x.  We just don't run `tsc`, so the latest driver causes this to fail.